src/components/UserDisplay.vue

```diff
@@ -19,20 +19,36 @@ export default {
   name: 'UserDisplay',          
     data() {
         return{
-        user: this.$store.state.user.currentUser,
+        user: {}
         }
     },
-  methods: {
-    getName(){
-      console.log(this.user)
-      if(!this.user.firstName || !this.user.lastName || this.user.firstName === "" || this.user.lastName === ""){
-        return "Placeholder Holdersen Place"
-      }
-      else{
-        return this.user.firstName + " " +  this.user.lastName
-      }
+  methods: {     
+    
+    
+     //getting the name from the user and making sure there is something in it
+    getName() {
+          this.user = this.$store.state.user.currentUser
+          
+          if(!this.user.firstName || !this.user.lastName || this.user.firstName === "" || this.user.lastName === "" || !this.user || this.user == {} ){
+            return "Placeholder Holdersen Place"
+          }
+          else{
+            return this.user.firstName + " " +  this.user.lastName
+       }
+    },
+
+    //getting picture for the avatar
+    getPicture() {
+      this.user = this.$store.state.user.currentUser
```

Comment:
Do this in 'created' step of the lifecycle.

Reply:
Remove the logic when setting dummy-data in data() function